core/mondoo-unix-openssl-vulnerability.mql.yaml

```diff
@@ -0,0 +1,88 @@
+# Read more about the policy structure at https://mondoo.com/docs/platform/policies/overview
+policies:
```

Comment:
We should include a general policy description since that shows up in the Policy Hub.